docs/development/getting-started.rst

```diff
@@ -84,6 +84,41 @@ Alternately, you can use our ``tox`` job:
 
     $ tox -e docs
 
+Building a Mac application
+~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+``mimic`` can be built using `py2app`_ and `pyobjc`_.
+Due to several quirks in the current version of `py2app`_, specific versions of
+libraries are needed in order for `py2app`_ to build correctly.
+These requirements are specified in ``py2app-requirements.txt``.
+
+`py2app`_ requires that a non-system python be installed for bundling.
+To get around this, I've found it easiest to install a brewed 2.7 python.
+This is preferable to using ``pyenv`` because ``brew`` installs a framework
+python.
```

Comment:
You were completely correct! It builds fine just using the system python. I'll change change the docs and the travis code to reflect this.